react-framework-master/server/index.js

```diff
@@ -16,22 +18,43 @@ app.get('/products/:id', function (req, res, next) {
 // app.listen(80, function () {
 //   console.log('CORS-enabled web server listening on port 3000')
 // })
-
-app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 
+app.use(bodyParser.urlencoded({ extended: false }));
+
+
+
 app.use(express.static(path.join(__dirname, "../client/dist")))
 
 app.listen(port, () => console.log(`Example app listening on port ${port}!`))
 
 app.get('/overview/:restaurantId', function (req, res) {
-	db.collection("restaurants").find( {"id": Number(req.params.restaurantId)}).toArray(function (err, result) {
```

Comment:
Just a general comment: are you converting tabs to spaces before pushing to github? Might help with formatting.